src/com/company/Power.java

```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class Power extends Service {
+
+    static int getPower(int Num, int Pow) {
```

Comment:
read about parameter names and change them
use camelCase
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Reply:
better use full name of the parameters

Reply:
this method will not work if you write getPower(3, -4)